Misc/Positions_Area Positions 3 UK Spares.txt

```diff
@@ -33,3 +33,4 @@ Scottish Control Galloway North:Scottish Control:121.370:STGN:S:STC:CTR:-:-:3601
 
 ;London Info
 London Information:London Information:125.470:LI:C:EGTT:CTR:-:-::
```

Comment:
Might as well do the .75 change here?

Reply:
Discussed in #4036 - @ChrisXPP and I have agreed not to make the change here as it would have clashed with his changes in #4044.

Reply:
@luke11brown resolving this as change is already covered elsewhere